README.md

```diff
@@ -314,6 +314,139 @@ jobs:
         imagepullsecrets: |
           demo-k8s-secret
 ```
+## Sample workflows with new environment variables ( which this action reads from ) which can be reused throughout the workflow and help with traceability fields.
+
+### End to end workflow for building and deploying container images 
+
+ - Env variable `REGISTRY_CREDENTIALS` list of credentials for container registry login as <registry_name><space><username><space><password>
+ - Env variable `DOCKERFILE_PATHS` is a list of dockerfile paths for images used as <image_name><space><dockerfile_path>
+
+```yaml
+on: [push]
+env:
+  REGISTRY_URL: contoso.azurecr.io
+  NAMESPACE: testnamespace1
+  REGISTRY_CREDENTIALS: |
+    contoso.azurecr.io ${ secrets.AZURE_CLIENT_ID } ${ secrets.AZURE_CLIENT_SECRET }
+  DOCKERFILE_PATHS: |
+    contoso.azurecr.io/k8sdemo:first ./Dockerfile

```

Comment:
I don't think this is a good design. 

Also take this change along with the PR.

Reply:
I agree it was bad design - I have reverted this particular change. We have already pushed the current changes to v1.2, so we should atleast update the README to reflect those changes - this is long overdue.
I will raise a new set of PRs with the design changes required and all other review comment changes, for master(with updated README), v1.2,v1.3 and v1.4.

Reply:
Why did you merge this PR? Given that you were changing the logic anyway?

Reply:
Like I mentioned in my above comment while merging, the changes had been discussed and pushed to v1.2 in code, and only the README update was pending for a long time. After discussing with Ambika, he suggested we go ahead with the merge and add the new changes as an incremental update, since it was pending discussion.